src/gui/src/Mixer/MixerSettingsDialog_UI.ui

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>MixerSettingsDialog_UI</class>
+ <widget class="QDialog" name="MixerSettingsDialog_UI">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>490</width>
+    <height>157</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form1</string>
```

Comment:
Could you remove this property? It will be overwritten in the .cpp file but get's listed for the translators who will have no idea what to do with it